databases/installing-postgresql.md

```diff
@@ -1,90 +1,131 @@
+
+BigSQL currently installs pgAdmin version 3 which is deprecated. It's best to choose EnterpriseDB which installs the latest version 4.
+
+<img src="images/step2.png" alt="step2">
+
+* **Step 3** - You will be prompted to desired Postgre version and operating system. Select the **latest Postgre version** and OS as per your environment. Click the **Download** Button.
+
+<img src="images/step3.png" alt="step3">
+
+* **Step 4** - Open the downloaded exe and Click next on the install welcome screen.
+
+<img src="images/step4.PNG" alt="step4">
+
+* **Step 5** - Change the Installation directory if required, else leave it to **default** and click **Next**.
+
+<img src="images/step5.PNG" alt="step5">
+
+* **Step 6** - You may choose the components you want to install in your system. You may uncheck Stack Builder & click **Next**.
+
+<img src="images/step6.PNG" alt="step6">
+
+* **Step 7** - You may change the data location & click **Next**.
+
+<img src="images/step7.PNG" alt="step7">
+
+* **Step 8** - Enter **super user password**. Make a note of it & click **Next**.
+
+<img src="images/step8.PNG" alt="step8">
+
+* **Step 9** - Leave the **port number default** & click **Next**.
+
+<img src="images/step9.PNG" alt="step9">
+
+* **Step 10** - Check the pre-installation summary & click **Next**.
+
+<img src="images/step10.PNG" alt="step10">
 
-- leave this lesson knowing this
-- and how to do this on their own
-- Even more things
-- Even more things
+* **Step 11** - Wait for installation.
 
-### Specific Things To Teach
+<img src="images/step11.PNG" alt="step11">
 
-- Focus on this specific talking point
-- and this
-	- This is a sub-thing about the thing
-- Even more things about the things
-- Even more things about the things
+* **Step 12** - Once install is complete you will see the Stack Builder prompt, uncheck that option(for advanced tools). click **Finish**.
 
-### Materials
+<img src="images/step12.PNG" alt="step12">
 
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
+* **Step 13** - To launch Postgre go to Start Menu and search pgAdmin 4.
 
-### Lesson
+<img src="images/step13.png" alt="step13">
 
-Here's text about introducing something and how it works.
+* **Step 14** - You will see pgAdmin homepage.
 
-- [Be sure to link the slideshow, video,](google.com)
-- [or other materials you expect them to follow as part of the lesson.](google.com)
+<img src="images/step14.png" alt="step14">
 
-Build on the first information. Have apprentices guess things, do an activity, etc.
+* **Step 15** - Click on Servers > Postgre SQL 11 in the left tree.
```

Comment:
Here, line 34,  74, and 131 you call it Postgre or Postgre SQL.  please change them to  PostgreSQL.

Reply:
done